src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -28,21 +28,17 @@ export const fetchingOrganizations = () => dispatch =>
     .then(payload =>
       dispatch(fetchOrganizations(payload)));
 
-export const fetchDictionaries = () => dispatch =>     
-api.dictionaries
-        .fetchingDictionaries()
-        .then(dispatch(isFetching(true)))  
-        .then(payload =>
-          dispatch(isSuccess(payload)),
-          dispatch(isFetching(false))
-        )
-        .catch((error) => () => {
-          if (error.payload){
-            dispatch(isSuccess(error.payload));
-            dispatch(isFetching(false));
-          } else {
-            dispatch(isErrored(error.payload.data));
-            dispatch(isFetching(false));
-          }
-        }
-        );
+export const fetchDictionaries = () => dispatch =>{
```

Comment:
I believe there should be a space between the brace and the arrow function on line 31

Reply:
That has been rectified.